examples/custom-theme/src/app.js

```diff
@@ -20,8 +20,8 @@
 
 import React, {Component} from 'react';
 import styled from 'styled-components';
-import window from 'global/window';
 import {connect} from 'react-redux';
+import AutoSizer from 'react-virtualized-auto-sizer';
```

Comment:
Not I see `AutoSizer` being use everywhere, It feels  like kepler.gl is married to it and can't leave without `AutoSizer`. Our example should be a simple one that requires a minimum amount of dependencies. Something like `AutoSizer` is good to have in `demo-app`, but probably not a must in all other examples.

I would rather remove it and use good old school ` window.addEventListener`